src/shapes/mesh.cpp

```diff
@@ -59,8 +60,8 @@ void Mesh::decodeTriangleIndexBytes(Span<const uint8_t> value) {
     BinaryReader reader(value);
     while (!reader.reachedEnd()) {
         uint64_t index = reader.readVarUint64();
-        assert(index < std::numeric_limits<uint16_t>::max());
-        buffer->push_back(index);
+        // TODO: if index *is* out of range, mark this mesh (and file?) as invalid
```

Comment:
We also then validate the indices here: https://github.com/rive-app/rive-cpp/blob/6617efb3be469c3362de5c8ea5831506f23a0fa0/src/shapes/mesh.cpp#L47-L51 so it may not be necessary to do it here.